apps/js-machine-front/src/app/stores/ui.store.ts

```diff
@@ -1 +1,13 @@
-export class UiStore {}
+import { action, observable, runInAction } from "mobx";
+import { RootStore } from "@js-machine-app/front/stores/root.store";
+
+export class UiStore {
+  @observable public isLoading?: boolean;
```

Comment:
Think that solution around single flag for all async calls is not the best option

Reply:
I suppose it's like isPageLoading, if separate loading flags is needed, we'll add that